OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/state/UserState.kt

```diff
@@ -0,0 +1,27 @@
+package com.onesignal.user.internal.state
+
+import org.json.JSONObject
+
+/**
+ * A user state.
+ */
+class UserState (
+        /**
+         * The unique identifier for OneSignal account. This will be an empty string
+         * until the user has been successfully logged in on the backend and
```

Comment:
If it will be an empty string when there is no onesignalId, what does a null `onesignalId` represent?